compose/.apps/murmur/murmur.yml

```
@@ -0,0 +1,21 @@
+services:
+  murmur:
+    container_name: murmur
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+      - MURMUR_SUPW=${MURMUR_SUPW}
```

```suggestion
      - MURMUR_SUPW=${MURMUR_SUPW}
      - PGID=${PGID}
      - PUID=${PUID}
      - TZ=${TZ}
```
Sorting alphabetically and this is good to go.